web/src/components/graph/areagraph.jsx

```diff
@@ -118,7 +118,11 @@ const AreaGraph = React.memo(
     const valueScale = useMemo(() => getValueScale(containerHeight, totalValues), [containerHeight, totalValues]);
     const datetimes = useCurrentZoneHistoryDatetimes();
     const startTime = datetimes.at(0);
-    const endTime = datetimes.at(-1);
+    const lastTime = datetimes.at(-1);
+    const intervalMs = datetimes.length > 1 ? lastTime.getTime() - datetimes.at(-2).getTime() : undefined;
+    // The endTime needs to include the last interval so it can be shown
+    const endTime = useMemo(() => new Date(lastTime.getTime() + intervalMs), [lastTime, intervalMs]);
+    const datetimesWithNext = [...datetimes, endTime];
```

Comment:
Because we render points at time T by drawing curves from T to T+1, we need to make sure the time scale goes to T+1, else the last curve won't be visible